superchain/superchain.go

```diff
@@ -449,17 +449,19 @@ type GenesisAccount struct {
 
 type Genesis struct {
 	// Block properties
-	Nonce      uint64  `json:"nonce"`
-	Timestamp  uint64  `json:"timestamp"`
-	ExtraData  []byte  `json:"extraData"`
-	GasLimit   uint64  `json:"gasLimit"`
-	Difficulty *HexBig `json:"difficulty"`
-	Mixhash    Hash    `json:"mixHash"`
-	Coinbase   Address `json:"coinbase"`
-	Number     uint64  `json:"number"`
-	GasUsed    uint64  `json:"gasUsed"`
-	ParentHash Hash    `json:"parentHash"`
-	BaseFee    *HexBig `json:"baseFeePerGas"`
+	Nonce         uint64  `json:"nonce"`
+	Timestamp     uint64  `json:"timestamp"`
+	ExtraData     []byte  `json:"extraData"`
+	GasLimit      uint64  `json:"gasLimit"`
+	Difficulty    *HexBig `json:"difficulty"`
+	Mixhash       Hash    `json:"mixHash"`
+	Coinbase      Address `json:"coinbase"`
+	Number        uint64  `json:"number"`
+	GasUsed       uint64  `json:"gasUsed"`
+	ParentHash    Hash    `json:"parentHash"`
+	BaseFee       *HexBig `json:"baseFeePerGas"`
+	ExcessBlobGas *uint64 `json:"excessBlobGas"` // EIP-4844
+	BlobGasUsed   *uint64 `json:"blobGasUsed"`   // EIP-4844
```

Comment:
We don't want `,omitempty` here?

Reply:
I should correct my comment above: the fields were not being decoded from the embedded `.json.gz` files committed to this repo and reexported from the Go module. 

I think `omitempty` only affects _encoding_ into JSON, so I don't think it will make any difference here?

Reply:
If we only ever decode, you're right. Just wanted to make sure we deliberately left this out.